src/renderer/screens/ColorPalette/PickerColorButton.js

```diff
@@ -0,0 +1,144 @@
+// -*- mode: js-jsx -*-
+/* Chrysalis -- Kaleidoscope Command Center
+ * Copyright (C) 2018, 2019  Keyboardio, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * This is Reactjs functional component that create button to choose colors from Color Picker
+ */
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import { SketchPicker } from "react-color";
+import paletteIcon from "./palette.svg";
+
+PickerColorButton.propTypes = {
+  classes: PropTypes.object.isRequired,
+  setColorFocusButton: PropTypes.func.isRequired,
+  colorFocusButton: PropTypes.object.isRequired,
+  disabled: PropTypes.bool.isRequired
+};
+
+const styles = theme => ({
+  root: {
+    position: "relative",
+    margin: "0 20px",
+    [theme.breakpoints.down("sm")]: {
+      marginLeft: 10,
+      marginRight: 0
+    }
+  },
+  swatch: {
+    padding: 5,
+    width: 40,
+    height: 40,
```

Comment:
Could we enlarge this to 60, please? See larger comment about resizing this entire palette.